src/utils/filterResults.ts

```diff
@@ -149,7 +151,13 @@ export default function filterResults (unsortedPropertyResults: UnprocessedResul
                 } else {
                     shouldAddProperty = shouldAddProperty || isUnderNeighbourLimit
                 }
-                if (shouldAddProperty) {
+
+                let toAddDuplicate: boolean = true
```

Comment:
I don't understand this name.

Reply:
Yeah, I guess it's suggesting to add a duplicate when what I want to do is not add a duplicate.